kotlin/internal/jvm/compile.bzl

```diff
@@ -554,9 +554,10 @@ def _run_kt_java_builder_actions(
     compile_jars = []
     output_jars = []
     kt_stubs_for_java = []
+    has_sources = srcs.kt or srcs.src_jars
```

Comment:
`has_kt_sources` might be a better name for this so that we don't mistakenly reference this for Java specific blocks of code.

Reply:
Addressed 👍🏼